domains/client/domain-operator/src/domain_bundle_producer.rs

```diff
@@ -24,14 +27,23 @@ use std::sync::Arc;
 use subspace_runtime_primitives::{Balance, BlockHashFor, ExtrinsicFor, HeaderFor};
 use tracing::info;
 
-/// Type alias for bundle header.
-pub type BundleHeaderFor<Block, CBlock> =
-    BundleHeader<NumberFor<CBlock>, BlockHashFor<CBlock>, HeaderFor<Block>, Balance>;
+/// Type alias for bundle header V0.
+pub enum BundleHeaderFor<Block: BlockT, CBlock: BlockT> {
+    V1(BundleHeaderV0<NumberFor<CBlock>, BlockHashFor<CBlock>, HeaderFor<Block>, Balance>),
+}
+
+/// Type alias for bundle header V1.
+pub type BundleHeaderV1For<Block, CBlock> =
+    BundleHeaderV0<NumberFor<CBlock>, BlockHashFor<CBlock>, HeaderFor<Block>, Balance>;
```

Comment:
<img width="474" height="267" alt="image" src="https://github.com/user-attachments/assets/5f57fa22-6602-4271-bb86-4496e97539d2" />

The different version numbers in these types need further explanation in comments (or fixing).